pages/instructorGradebook/instructorGradebook.sql

```diff
@@ -1,26 +1,38 @@
 -- BLOCK course_assessments
 SELECT
-    a.id AS assessment_id,a.number AS assessment_number,
+    a.id AS assessment_id,
+    a.number AS assessment_number,
```

Comment:
While I appreciate the effort to clean up code, please don't introduce additional unrelated changes (such as code reformatting) into bug fix PRs. It makes it hard for reviewers to see what's actually changed in the PR.